sacrebleu/tokenizers/tokenizer_ja_mecab.py

```diff
@@ -1,20 +1,33 @@
 # -*- coding: utf-8 -*-
-
-import MeCab
+try:
+    import MeCab
+    import ipadic
+except ModuleNotFoundError:
+    # Don't fail until the tokenizer is actually used
+    MeCab = None
 
 from .tokenizer_none import NoneTokenizer
 
+FAIL_MESSAGE = """
+Japanese tokenization requires extra dependencies, but you do not have them installed.
+Please install them like so.
+
+    pip install sacrebleu[ja]
+"""
 
 class TokenizerJaMecab(NoneTokenizer):
     def __init__(self):
-        self.tagger = MeCab.Tagger("-Owakati")
+        if MeCab is None:
+            raise RuntimeError(FAIL_MESSAGE)
+        self.tagger = MeCab.Tagger(ipadic.MECAB_ARGS + " -Owakati")
 
         # make sure the dictionary is IPA
         # sacreBLEU is only compatible with 0.996.5 for now
```

Comment:
Can you remove these comments related to 0.996.5 then?

Reply:
Also, can you rebase your branch as master moved forward by 1 commit in the meanwhile?

Reply:
@ozancaglayan I think you can merge master into this polm:japanese-fix branch yourself, e.g. by clicking on the "Update branch" button on GitHub. @mjpost prefers to squash all commits in the PR, so in the end it does not matter if you rebase the PR or merge in master.

Reply:
I removed the comments and updated the branch.